packages/create-block/lib/templates/es5/readme.txt.mustache

```diff
@@ -0,0 +1,54 @@
+=== {{title}} ===
+Contributors:      {{author}}
+Tags:              comments, spam
+Requires at least: 5.0
+Tested up to:      5.3.2
+Stable tag:        {{version}}
+Requires PHP:      7.0.0
```

Comment:
From https://wordpress.org/about/requirements/:

> Note: If you are in a legacy environment where you only have older PHP or MySQL versions, WordPress also works with PHP 5.6.20+ and MySQL 5.0+, but these versions have reached official End Of Life and as such may expose your site to security vulnerabilities.

I guess 7.0.0 is a good idea here 👍